esp-hal/src/gpio/etm.rs

```diff
@@ -127,6 +127,7 @@ impl<'d> Channels<'d> {
 }
 
 /// Configuration for an ETM controlled GPIO input pin
+// TODO: remove this
```

Comment:
Is the TODO intentional?

Reply:
yes, this is now redundant with gpio::InputConfig, just out of scope